voxel51/users/api.py

```diff
@@ -458,6 +458,30 @@ def delete_data(self, data_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    def batch_update_data_ttl(self, data_ids, days):
+        '''Updates the expiration date of all specified data.
+
+        Args:
+            data_ids (list): the data IDs
+            days (float): the number of days by which to extend the lifespan
+                of the data
+
+        Raises:
+            :class:`APIError` if the request was unsuccessful
+        '''
+        return self._batch_request("data", "ttl", data_ids, {"days": days})
```

Comment:
I believe `return` should be deleted for all of these since their non-batch counterparts also don't return anything

Reply:
This returns statuses of every item (e.g. whether deleting each item succeeded or not). There's not really another way to get that information.

Reply:
ah got it. so the batch methods are "smart" and will perform the action on whichever items it can without blowing up the whole request on a single failure? makes sense

Reply:
We could potentially improve the error handling here, though - the console will display all of the errors for items where an action failed, but this currently relies on the API user to do that.

Reply:
I'm fine with it as is. It's clear whose responsibility it is to deal with failures.